integration-tests/src/test/resources/performance-features/send-cloud-events.feature

```diff
@@ -65,7 +65,7 @@ Feature: Sending cloud events performance tests
           increasingRate:
             initialUsersPerSec: 1
             targetUsersPerSec: 10
-            maxSessions: 10
+            maxSessions: 20
```

Comment:
Is this related to upload to Horreum?

Reply:
No, I have just noticed that some tests executed locally crashed on lack of sessions, so adding it here.